src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -0,0 +1,77 @@
+"""
+File       : MSUnmerged.py
+
+Description:
+
+This MicroService is meant to remove all the files from the unmerged area of the.
```

Comment:
Just to avoid misinterpretations, I would suggest to rephrase it to something like '...remove all the no longer needed files ...'

Reply:
Agreed

Reply:
done